scripts/get_nodejs_bin

```diff
@@ -9,9 +9,9 @@ NODEJS_LINUX_x64="linux-x64"
 NODEJS_LINUX_x86="linux-x86"
 NODEJS_MAC_x64="darwin-x64"
 # TODO : This is temporal fixed version. Need to get latest or specific version from dist repository
-NODEJS_VERSION="v6.9.5"
-NODEJS_VERSION_NAME="node-$NODEJS_VERSION"
-NODEJS_DOWNLOAD_URL="https://nodejs.org/dist/$NODEJS_VERSION/"
+NODEJS_VERSION="6.9.5"
+NODEJS_VERSION_NAME="node-v$NODEJS_VERSION"
+NODEJS_DOWNLOAD_URL="https://nodejs.org/dist/v$NODEJS_VERSION/"
```

Comment:
I think we need to delete last slash because of redundancy.

Plz check below line 37.
target_url="$NODEJS_DOWNLOAD_URL/$target_node_archive_file"